cms/templates/partials/megamenu/megamenu.html

```diff
@@ -11,7 +11,8 @@
     {#  Main menu conatiner - start #}
     <div class="main-menu-wrapper nhsuk-grid-column-three-quarters">
       <div class="main-menu">
-        <div><div class="programme_section">Programmes</div>{% main_menu max_levels=1 template="partials/megamenu/navigation.html" %}</div>
+        <div><div class="programme_section">Programmes</div>
```

Comment:
I'm not completely happy with "programmes" being a title here, might need to go another round with content design to see if this means anything for site users, but leave it for now.